### PR TITLE
Add a basic kas config

### PR DESCRIPTION
**Pull request by graugans**
kas is a set-up tool for bitbake projekts. It simplifies the local.conf
and bblayer file handling. It also provides the possibility to include
configurations from other layers.

Signed-off-by: Christian Ege <ch@ege.io>

### CODE REVIEW

**Comment by graugans** [comment]
I hope this is useful. With kas it is also easier to support for example `meta-swupdate` this could be handled like `kas-poky-meson-swu.yml` and for rauc `kas-poky-meson-rauc.yml` if you prefer a more clean project dir something like a `kas` subfolder could be used.

**Comment by superna9999** [comment]
Thanks a lot !

I’m new to kas but i know it’s the new tool en vogue, mender talked about using kas for the community layers.

other layers seems to have a kas subdirectory, so I think it would be best indeed, if you move to a subdir and update readme accordingly I’ll merge so we can go forward on the subject !

File: kas/kas-poky-meson.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+header:
+  version: 8
+
+machine: amediatech-x96-max
+distro: poky
+target:
+  - amlogic-image-headless-sd
+
+repos:
+  meta-raspberry:
```
Comment (superna9999) [inline kas/kas-poky-meson.yml:10]: Why is there reference to meta-raspberry ? (Perhaps it’s needed, no worry then)

Comment (graugans) [reply kas/kas-poky-meson.yml:10]: Oh, that was a copy paste. I'll fix it.

**Pull request merged**